### DISCUSSION TITLE
Why lottie-ios 4.4.3 require iOS 13.0

### DISCUSSION THREAD

**Discussion by SunshineTraveller** [airbnb/lottie-ios#2384] [General]
Because of Apple SDK Privacy mainfest requirements, our project use:
pod 'lottie-ios', '4.4.3'
after I execute 'pod install', I got the error below:

[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In Podfile:
    TGPKit (from `https://git.100tal.com/tal_znxx_family/genie_tpgkit_ios`, branch `feature/zhanglimin/1.25.0_privacy`) was resolved to 0.0.1, which depends on
      lottie-ios (= 4.4.3)
Specs satisfying the `lottie-ios (= 4.4.3)` dependency were found, but they required a higher minimum deployment target.

After I download and unzip the lottie-ios-master.zip file, I found : s.ios.deployment_target = '13.0' in the lottie-ios.podspec. The mimimum Deployments version of XCode 15.3 is iOS 12.0, why lottie-ios 4.4.3 require iOS 13.0?
Is the only way to solve this is to modify our Deploayments to iOS 13.0? 

<img width="572" alt="截屏2024-04-16 14 44 42" src="https://github.com/airbnb/lottie-ios/assets/18361009/a4c0bbf0-fb09-4635-8c5e-b705adf6c588">

**Comment by calda** [comment]
Lottie now requires iOS 13+. If you need to support iOS 12, you could continue using Lottie 4.4.1 (which does include the privacy manifest).

**Reply by SunshineTraveller** [reply]
All right
